db/seeds.rb

```diff
@@ -7,812 +7,176 @@
-StoreItem.create({ name: 'Drill Bit', price: 1, quantity: 1000})
-StoreItem.create({ name: 'Drop Cloth', price: 2, quantity: 1000})
-StoreItem.create({ name: 'Electrical Box - 1 Gang', price: 1, quantity: 1000})
-StoreItem.create({ name: 'Electrical Box - 2 Gang', price: 1, quantity: 1000})
-StoreItem.create({ name: 'Electrical Box - 4 in Round', price: 3, quantity: 1000})
-StoreItem.create({ name: 'Electrical Box - 4 in Square', price: 3, quantity: 1000})
-StoreItem.create({ name: 'Electrical Box Cover - 4 in Round', price: 2, quantity: 1000})
-StoreItem.create({ name: 'Electrical Box Cover - 4 in Square', price: 2, quantity: 1000})
-StoreItem.create({ name: 'Joist Hander - 2x6', price: 2, quantity: 1000})
-StoreItem.create({ name: 'Joist Hander - 2x8', price: 2, quantity: 1000})
-StoreItem.create({ name: 'Light Switch', price: 3, quantity: 1000})
-StoreItem.create({ name: 'Light Switch Cover', price: 1, quantity: 1000})
-StoreItem.create({ name: 'Outlet', price: 3, quantity: 1000})
-StoreItem.create({ name: 'Outlet Cover - 1 Gang', price: 1, quantity: 1000})
-StoreItem.create({ name: 'Outlet Cover - 2 Gang', price: 1, quantity: 1000})
-StoreItem.create({ name: 'Plug - Angle', price: 12, quantity: 1000})
-StoreItem.create({ name: 'Plug - Straight', price: 12, quantity: 1000})
-StoreItem.create({ name: 'Romex 12/2 - 10ft', price: 5, quantity: 1000})
-StoreItem.create({ name: 'Screw Bit - Phillips (x2)', price: 1, quantity: 1000})
-StoreItem.create({ name: 'Screws (x10)', price: 1, quantity: 1000})
-StoreItem.create({ name: 'Spade Bit - 3/4"', price: 5, quantity: 1000})
-StoreItem.create({ name: 'Staple Gun Staples (x4 Rows)', price: 1, quantity: 1000})
-StoreItem.create({ name: 'Tarp', price: 15, quantity: 1000})
-StoreItem.create({ name: 'Wire Nuts (x10)', price: 1, quantity: 1000})
-StoreItem.create({ name: 'Wire Staples (x20)', price: 1, quantity: 1000})
+puts '  FAQs'
+
+csv_text = File.read(Rails.root.join('lib', 'seeds', gdrive_doc + 'faqs.csv'))
+csv = CSV.parse(csv_text, :headers => true)
+csv.each do |row|
+  Faq.create(question: row['question'].strip, answer: row['answer'].strip)
+end
+
+puts '  Tasks'
+
+csv_text = File.read(Rails.root.join('lib', 'seeds', gdrive_doc + 'tasks.csv'))
+csv = CSV.parse(csv_text, :headers => true)
+csv.each do |row|
+  Task.create(name: row['name'].strip, description: row['description'].strip, due_at: DateTime.strptime(row['due_at'], '%m/%d/%Y %H:%M:%S'))
+end
+
+puts '  Shifts'
+
+csv_text = File.read(Rails.root.join('lib', 'seeds', gdrive_doc + 'shifts.csv'))
+csv = CSV.parse(csv_text, :headers => true)
+csv.each do |row|
+  organization = Organization.find_by_name(row['organization'].strip)
+  if !organization
+    puts '    Organization (' + row['organization'].strip + ') does not exist'
+    fail
+  end
+
+  shift_type ||= ShiftType.find_by_name(row['shift_type'].strip)
+  shift_type ||= ShiftType.create(name: row['shift_type'].strip)
```

Comment:
Just as an FYI, since this creates the shift types in the order they appear in `lib/seeds/2018-seeds - shifts.csv`, rather than the old canonical order, the various scopes for viewing shifts in Binder are quite broken.  Obviously, we're already into production for this year, but #275 seeks to address this if you'd like to bring it in as a hotfix.